### DISCUSSION TITLE
Get top 5 tickers by sector

### DISCUSSION THREAD

**Discussion by kimbau-dev** [ranaroussi/yfinance#2047] [Q&A] [answered]
Hello, is there a way to get the top 5 tickers by market cap in a sector, by adding calling a function that accepts sector as a parameter.

**Comment by ericpien** [comment, marked as answer]
Hi @f-bautista, you can query it in the dev branch now with
```python
import yfinance as yf
tech = yf.Sector('technology')
tech.top_companies
```

Related PR: https://github.com/ranaroussi/yfinance/pull/2058
